tests/app/Models/SearchTest.php

```diff
@@ -166,14 +168,14 @@ public static function test__construct_whenInputContainsPubdate_setsPubdateValue
 	 * @return array<array<mixed>>
 	 */
 	public static function providePubdateSearch(): array {
-		return array(
-			array('pubdate:2007-03-01T13:00:00Z/2008-05-11T15:30:00Z', 1172754000, 1210519800),
-			array('pubdate:2007-03-01T13:00:00Z/P1Y2M10DT2H30M', 1172754000, 1210519799),
-			array('pubdate:P1Y2M10DT2H30M/2008-05-11T15:30:00Z', 1172754001, 1210519800),
-			array('pubdate:2007-03-01/2008-05-11', strtotime('2007-03-01'), strtotime('2008-05-12') - 1),
-			array('pubdate:2007-03-01/', strtotime('2007-03-01'), null),
-			array('pubdate:/2008-05-11', null, strtotime('2008-05-12') - 1),
-		);
+		return [
+			['pubdate:2007-03-01T13:00:00Z/2008-05-11T15:30:00Z', 1172754000, 1210519800],
+			['pubdate:2007-03-01T13:00:00Z/P1Y2M10DT2H30M', 1172754000, 1210519799],
+			['pubdate:P1Y2M10DT2H30M/2008-05-11T15:30:00Z', 1172754001, 1210519800],
+			['pubdate:2007-03-01/2008-05-11', strtotime('2007-03-01'), strtotime('2008-05-12') - 1],
+			['pubdate:2007-03-01/', strtotime('2007-03-01'), null],
+			['pubdate:/2008-05-11', null, strtotime('2008-05-12') - 1],
+			];
```

Comment:
```suggestion
		];
```

Plenty of wrongly indented arrays. Please check them all

Reply:
There were too many errors in this file, so I just autoformatted the arrays https://github.com/FreshRSS/FreshRSS/pull/6748/commits/52c74d324b58d6762319b2ad6631879662eeca8e